source/aws_solutions/qnabot/cli/qnabot_cli_helper.py

```diff
@@ -0,0 +1,446 @@
+    """
+
+    try:
+        s3_client = get_service_client("s3")  # boto3.client('s3')
+        # get object only if the object has changed since last request
+        response = s3_client.get_object(
+            Bucket=bucket, Key="status/" + os.path.basename(export_filename), IfModifiedSince=exportdatetime
+        )
+
+        obj_status_details = json.loads(response["Body"].read().decode("utf-8"))  # read object body
+
+        return_response = {"status": obj_status_details["status"], "error_code": "none"}
+        return_response = json.dumps(return_response, indent=4)
+        return return_response
+    except ClientError as err_exception:
+        # if object has not been modified (304) or the object is not available in S3 bucket yet (NoSuchKey)
+        if err_exception.response["Error"]["Code"] in ("304", "NoSuchKey"):
+            return error_response(
+                error_code=err_exception.response["Error"]["Code"],
+                message=err_exception.response["Error"]["Message"],
+                comments="Please note: Export processing may take longer to process "
+                + "depending on the number of questions.",
+                status="Pending",
+                show_error=False,
+            )
+        return error_response(
+            error_code=err_exception.response["Error"]["Code"],
+            message=err_exception.response["Error"]["Message"],
+            comments="",
+            status="Error",
+            show_error=True,
+        )
+
+
+def convert_json_to_jsonl(source_filename: str):
+    """
+    Convert to JSON Lines format
+    :param source_filename: import directory and filename
+    :return: file contents
+    """
+
+    try:
+        with open(source_filename, "rb") as obj_file:  # open file in read mode
+            str_file_contents = obj_file.read()  # read from file
+        try:
+            str_file_contents = json.loads(str_file_contents)
+            str_lines = ""
+            for entry in str_file_contents["qna"]:
+                str_lines = str_lines + json.dumps(entry) + "\n"
+            return str_lines
+        except json.decoder.JSONDecodeError as err_exception:
+            return error_response(
+                error_code="",
+                message=err_exception.msg,
+                comments="There was an error reading the file "
+                + source_filename
+                + ". Check the file format and try again",
+                status="Error",
+                show_error=True,
+            )
+        except UnicodeDecodeError as err_exception:
+            return error_response(
+                error_code="",
+                message=err_exception.__doc__,
+                comments="There was an error reading the file "
+                + source_filename
+                + ". Check the file format and try again",
+                status="Error",
+                show_error=True,
+            )
+        except TypeError as err_exception:
```

Comment:
Handling of UnicodeDecodeError and TypeError seems exactly the same. Are they? If so, we can catch both exception together and them together, namely something like

`except (UnicodeDecodeError, TypeError) as error_exception:`